mrkingz/cmrc
# #168432360 add email verification

#### What does this PR do?
- implements email verification
- implements account confirmation
 #### Description of Task to be completed?
- add implements email verification
- add implements account confirmation
 #### How should this be manually tested?
```
Step 1:
 - clone the repo
 - cd into cmrc
 - run 'npm install' or' yarn add'
 - run 'npm run build-dev'
 -  navigate to '/api/v1/auth/signup on postman'
 - enter a value for the following fields:
 firstName, lastName, email and password

Step 2:
- Go to your email, click the confirm button attached the mail
- Copy the url displayed on the browsers tab
- paste it in your postman tab and send
```
 #### What are the relevant pivotal tracker stories?
[#168431360](https://www.pivotaltracker.com/story/show/168431360)
